### ISSUE TITLE
README.rst contains non-ASCII characters fails installtion

### ISSUE THREAD

**Issue by NateScarlet** [jeffknupp/sandman2#93]
```powershell
> pip install sandman2
Collecting sandman2
  Using cached https://files.pythonhosted.org/packages/34/43/65317a5a01c16d494a68b37bc21d9cbe17c3fd089b76835fdbda60f1973b/sandman2-1.2.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\34357\AppData\Local\Temp\pip-install-1toj7xsk\sandman2\setup.py", line 19, in <module>
        LONG_DESCRIPTION = read('README.rst')
      File "C:\Users\34357\AppData\Local\Temp\pip-install-1toj7xsk\sandman2\setup.py", line 17, in read
        return codecs.open(os.path.join(HERE, *parts), 'r').read()
    UnicodeDecodeError: 'gbk' codec can't decode byte 0xa6 in position 2084: illegal multibyte sequence
```

# Solution1

README.rst contains non-ASCII character `’`, should replace it with ASCII character `'`
See https://github.com/pypa/virtualenv/issues/201#issuecomment-3145690

# Solution2

https://github.com/jeffknupp/sandman2/blob/1ce21d6f7a6df77fa96fab694b0f9bb8469c166b/setup.py#L16-L17

Intentionally *do* adding an 'utf-8' encoding option to open

**Comment by Expire0** [comment]
I submitted a pull request for this and hopefully we can get it merged.  The installation continued to fail until i added the encoding argument.  Pull request https://github.com/jeffknupp/sandman2/pull/99

**Comment by jeffknupp** [comment]
Fixed by #99
